tedana/gscontrol.py

```diff
@@ -137,77 +137,67 @@ def gscontrol_mmix(optcom_ts, mmix, mask, comptable, ref_img, out_dir='.'):
     Filename                  Content
     ======================    =================================================
     sphis_hik.nii             T1-like effect
-    hik_ts_OC_T1c.nii         T1-corrected BOLD (high-Kappa) time series
-    dn_ts_OC_T1c.nii          Denoised version of T1-corrected time series
-    betas_hik_OC_T1c.nii      T1 global signal-corrected components
-    meica_mix_T1c.1D          T1 global signal-corrected mixing matrix
+    hik_ts_OC_MIR.nii         T1-corrected BOLD (high-Kappa) time series
+    dn_ts_OC_MIR.nii          Denoised version of T1-corrected time series
+    betas_hik_OC_MIR.nii      T1 global signal-corrected components
+    meica_mix_MIR.1D          T1 global signal-corrected mixing matrix
     ======================    =================================================
     """
-    LGR.info('Performing T1c global signal regression to remove spatially '
-             'diffuse noise')
-    RepLGR.info("T1c global signal regression was then applied to the "
-                "data in order to remove spatially diffuse noise.")
+    LGR.info("Performing minimum image regression to remove spatially-diffuse noise")
+    RepLGR.info(
+        "Minimum image regression was then applied to the "
+        "data in order to remove spatially diffuse noise."
+    )
 
     all_comps = comptable.index.values
-    acc = comptable[comptable.classification == 'accepted'].index.values
-    ign = comptable[comptable.classification == 'ignored'].index.values
+    acc = comptable[comptable.classification == "accepted"].index.values
+    ign = comptable[comptable.classification == "ignored"].index.values
     not_ign = sorted(np.setdiff1d(all_comps, ign))
 
     optcom_masked = optcom_ts[mask, :]
     optcom_mu = optcom_masked.mean(axis=-1)[:, np.newaxis]
     optcom_std = optcom_masked.std(axis=-1)[:, np.newaxis]
 
-    """
-    Compute temporal regression
-    """
-    data_norm = (optcom_masked - optcom_mu) / optcom_std
-    cbetas = lstsq(mmix, data_norm.T, rcond=None)[0].T
-    resid = data_norm - np.dot(cbetas[:, not_ign], mmix[:, not_ign].T)
+    # Compute temporal regression
+    optcom_z = stats.zscore(optcom_masked, axis=-1)
+    comp_pes = np.linalg.lstsq(mmix, optcom_z.T, rcond=None)[0].T
+    resid = optcom_z - np.dot(comp_pes[:, not_ign], mmix[:, not_ign].T)
 
-    """
-    Build BOLD time series without amplitudes, and save T1-like effect
-    """
-    bold_ts = np.dot(cbetas[:, acc], mmix[:, acc].T)
-    t1_map = bold_ts.min(axis=-1)
+    # Build BOLD time series without amplitudes, and save T1-like effect
+    mehk_ts = np.dot(comp_pes[:, acc], mmix[:, acc].T)
```

Comment:
It's a little counter-intuitive to me to change this from "bold_ts" to "mehk_ts" (mid-elbow, high-kappa?) but leaving the comment the same. I'd recommend either changing the comment or reverting the variable name change.

Reply:
You're totally right. I will improve the comment. I switched to `MEHK` (multi-echo high kappa) because "BOLD" is so unclear when we're dealing with multi-echo data.

Reply:
Thanks, I think that helps !! On the same point : can we write out MEHK here ?